Classes/Site.php

```diff
@@ -162,6 +170,19 @@ public static function getAvailableSites()
     }
 
     /**
+     * Returns the first available Site.
+     *
+     * @param boolean $stopOnInvalidSite
```

Comment:
The intension is if the Exception from the constructor is bubbled up or not. Normally i would say it is good the the exception get's thrown, but when getting the collection in a loop it breaks the look, which is not allways wanted. Sometimes you just want to skip this item, which is the default now.